docs/ref/contrib/staticfiles.txt

```diff
@@ -333,40 +338,6 @@ simply specify a custom entry in the :setting:`CACHES` setting named
 
 .. currentmodule:: django.contrib.staticfiles.templatetags.staticfiles
```

Comment:
This should be removed as it was in my patch. Did you apply the edits manually such that I should check them line by line?